sdk/core/azure-core-serializer-json-jackson/src/main/java/com/azure/core/serializer/json/jackson/implementation/School.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
it seems some test models got into main folder

Reply:
Good catch, I was wondering why linting was failing on this, it was that they were in main